variation-commons-mongodb/src/main/java/uk/ac/ebi/eva/commons/mongodb/repositories/VariantRepositoryImpl.java

```diff
@@ -108,6 +108,13 @@ public Long countByRegionsAndComplexFilters(List<Region> regions, List<VariantRe
                 .distinct(VariantMongo.CHROMOSOME_FIELD));
     }
 
+    @Override
+    public List<VariantMongo> findByChromosomeAndOtherBeaconFilters(String chr, List<VariantRepositoryFilter> filters) {
```

Comment:
The current API allows to user to make mistakes, in the sense that they could request a precise (`start`) and imprecise (`startMin` + `startMax`) coordinate at the same time. I would recommend following the interface of the `xxxByRegionAndComplexFilters` as much as possible, maybe like this:

```java
public List<VariantMongo> findByRegionAndBeaconFilters(Region startRange, Region endRange, List<VariantRepositoryFilter> filters)
```

When `start` is provided as a parameter in the URL, the `startRange` object will have the same pair of coordinates. This makes easy to check on the REST side if too many ranges have been set.